Kinan/14-PB-datastructure-array-II.js

```diff
@@ -0,0 +1,31 @@
+let euroCities = 
+[
+    "Paris",
+    "London",
+    "Valletta",
+    "Prague",
+    "Rome"
+]
+euroCities[0]= "Berlin";
+console.log(euroCities);             //[ 'Berlin', 'London', 'Valletta', 'Prague', 'Rome' ]
+console.log(euroCities.length); //5
+
+euroCities= euroCities.splice(0, (euroCities.length-1));
+console.log(euroCities);       
+ // [ 'Berlin', 'London', 'Valletta', 'Prague' ]
+
+euroCities.push("Budapest");
+console.log(euroCities);     
+//[ 'Berlin', 'London', 'Valletta', 'Prague', 'Budapest' ]
+let euroCities2= euroCities.splice(1,2); 
+console.log(euroCities);
+// [ 'Berlin', 'Prague', 'Budapest' ]
+let asianCities = [
```

Comment:
please check this article to learn more about clone array

https://stackoverflow.com/questions/3978492/fastest-way-to-duplicate-an-array-in-javascript-slice-vs-for-loop